frontend/src/dto/ApplyClientNumberDto.ts

```diff
@@ -26,8 +26,9 @@ export interface FormDataDto {
     incorporationNumber: string;
     businessName: string;
     goodStandingInd: string;
-    birthDate: string;
+    birthdate: string;
```

Comment:
Use `java.time.LocalDate` instead of `String`

Reply:
This is in TypeScript :)